packages/backend/src/rates/util.ts

```diff
@@ -11,9 +11,27 @@ export interface Asset {
   scale: number
 }
 
-export function convert(opts: ConvertOptions): bigint {
+export interface ConvertResults {
+  amount: bigint
+  scaledExchangeRate: number
+}
```

Comment:
I preferred to return the `scaledExchangeRate` here instead of re-calculating in local `getQuote` because:

1. it should always be the same and duplicating the calculation creates opportunity to diverge. 

2. it was kind of a lot of code duplication and kinda specific stuff I was doing for each conversion case `debitAmount`, `receiver.incomingAmount`, `receiveAmount`.  just felt like this was cleaner.

I mention it though because it did require updating the type many places where we didnt otherwise care about it.